amazon_price_per_item.user.js

```diff
@@ -966,6 +1017,11 @@
       );
     });
 
+      regExes.push(new RegExp(
```

Comment:
I think it would be better to allow `RegExp` objects in the `qualifiers` array.

Reply:
may be, I was not sure abou the right strategy.
I tested sometihng like this t first, but found this way afterall.
Feel free to adapt ;)